bin/ingest-csv

```diff
@@ -14,7 +14,6 @@ end
 if filename.nil?
   $stderr.puts "Usage: #{$PROGRAM_NAME} <csv file>"
   $stderr.puts "Usage: #{$PROGRAM_NAME} <csv file> <image directory> <object type>"
-  $stderr.puts "Usage: #{$PROGRAM_NAME} <csv file> <image directory>"
```

Comment:
You can use the old data files if you also pass the class as the third argument.